modules/sdcore-control-plane-k8s/terraform.tf

```diff
@@ -5,7 +5,7 @@ terraform {
   required_providers {
     juju = {
       source  = "juju/juju"
-      version = "~> 0.10.1"
+      version = ">= 0.10.1"
```

Comment:
Could we write  ~> 0.11.0 ?

Reply:
Ups, it should be `~> 0.11.0`... Looks like I accidentally pushed my test change...

Reply:
Fixed.